core/model/modx/sources/modfilemediasource.class.php

```diff
@@ -1134,11 +1134,9 @@ public function getObjectsInContainer($path) {
                     'id' => $bases['urlAbsoluteWithPath'].$fileName,
                     'name' => $fileName,
                     'cls' => 'icon-'.$fileExtension,
-                    'file_width' => $size[0],
-                    'file_height' => $size[1],
```

Comment:
Could you explain why you're getting rid of these array items?

Reply:
I introduced them with my PR #13629. So they occurred first in 2.6.

My idea was to have separate variables for a) the width and height of the preview image in the MODX media browser (image_width and image_height) and b) for the display of the original image's size (file_width and file_height) displayed below the preview image.

But in my aforementioned PR image_width and image_height is the size of the with phpthumb processed (and therefor often downscaled) image.

My error was that I didn't take in account that other extras like ImagePlus rely on and process image_width and image_height.

So I basically only reverted image_width and image_height to the original state and therefor file_width and file_height are obsolet.

Reply:
Or in short: I changed a variables inital purpose and this reverts it.

Reply:
@Mark-H Did this answer your question?

If no, anything else that I can clear up?

If yes, may I suggest considering this for 2.6.1 or the next patch? It's confirmed here #13670 and since I'm also responsible for the PR #13629 I can secure that there will be no problems with the new Media Browser features introduced with 2.6.